### ISSUE TITLE
Miou

### ISSUE THREAD

**Issue by long123524** [ggsDing/ASLNet#4]
Good work. But I have a problem, When describing the pixel-level evaluation indicator, such as  mIou, you use the IOU representation, are the two the same in the paper?
![image](https://user-images.githubusercontent.com/73996375/183023460-99bc005d-d844-4609-961d-eaef02f54c21.png)

**Comment by DingLei14** [comment]
Yes. mIoU is just the mean of the positive IoU and the negative IoU.
